airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -578,9 +578,9 @@ definitions:
           - "created_at"
           - "{{ config['record_cursor'] }}"
       datetime_format:
-        title: Cursor Field Datetime Format
+        title: Outgoing Datetime Format
         description: |
-          The datetime format of the Cursor Field. Use placeholders starting with "%" to describe the format the API is using. The following placeholders are available:
+          The datetime format used to format the datetime values that are sent in outgoing requests to the API. Use placeholders starting with "%" to describe the format the API is using. The following placeholders are available:
```

Comment:
Should it be "The datetime format **is** used to format..." ?

Reply:
I think either one would work, the way I had it was following the grammar of the original description, where the first sentence is more of a descriptor rather than a full sentence